src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs

```diff
@@ -21,75 +22,88 @@ public CentOpsFixture(IConfiguration configuration, TestClients testClients)
 
             // Setup
             _configuration = configuration;
-            TestInstitutionName = $"TestInstitution{DateTime.UtcNow.ToString("yyyyMMddHHmmss", CultureInfo.CurrentCulture)}";
-            _institutionsUri = new Uri($"{_configuration["CentOpsUrl"]}/admin/institutions");
-            _participantsUri = new Uri($"{_configuration["CentOpsUrl"]}/admin/participants");
-            _testClients = testClients;
+            _useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.CurrentCulture);
 
-            // Create Institution
-            var institutionPostBody = JsonSerializer.Serialize(new InstitutionRequest()
+            if (_useFixture)
             {
-                Name = TestInstitutionName,
-            });
-            var institution = _testClients.CentOpsAdminClient.Request<Institution>(Verb.Post, _institutionsUri, institutionPostBody).Result;
+                var uniqueTestId = DateTime.UtcNow.ToString("yyyyMMddHHmmss", CultureInfo.CurrentCulture);
```

Comment:
Please use `InvariantCulture`:
```suggestion
                var uniqueTestId = DateTime.UtcNow.ToString("yyyyMMddHHmmss", CultureInfo.InvariantCulture);
```

Reply:
Fixed, thanks